capitalizedecorator.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'basedecorator'
+
+class CapitalizeDecorator < BaseDecorator
+  attr_accessor :name
+
+  def initialize(nameable)
+    super(nameable)
+    @name = nameable.name
+  end
+
+  def correct_name
+    @name = @name.upcase
+    @name
+  end
```

Comment:
- [ ] @Katended Great work in your CapitalizeDecorator however the correct_name is supposed to Capitalize the first letter of the given name and not capitalize all letters.